README.md

```diff
@@ -1886,31 +1886,31 @@ $ git signatures show v1.0.0
  ```
 
 ## [git-profile](https://github.com/dm3ch/git-profile-manager)
-
-```
 # add a work profile
+```
 $ git profile add work
 Name: Name Surname
 Email: name@work-domain.com
 Signing Key:
 Profile work added successfully
+```
 
 # add a personal profile
+```
 $ git profile add home -n "Name Surname" -e name@gmail.com
 Profile home added successfully
+```
 
 # list out our saved profiles
+```
 $ git profile list
 Existing profiles:
 work
 home
+```
 
 # set the current git repository user to the home profile
+```
 $ git profile use work
-# Currently there will be no output in case of success
-
-## License
```

Comment:
Could you add the license back?

Reply:
Added real license info.

License that was in previous RP was just copy-pasted.